solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -135,17 +137,25 @@ contract WalletRegistry is Ownable {
         maliciousDkgResultNotificationRewardMultiplier = 100;
 
         dkg.init(_sortitionPool, _dkgValidator);
+        dkg.setSeedTimeout(1440); // ~6h assuming 15s block time // TODO: Verify value
         dkg.setResultChallengePeriodLength(11520); // ~48h assuming 15s block time
         dkg.setResultSubmissionTimeout(100 * 20); // TODO: Verify value
         dkg.setSubmitterPrecedencePeriodLength(20); // TODO: Verify value
     }
 
+    // TODO: Update to governable params
```

Comment:
When implementing governance mechanism for it, please do not apply a delay.

Reply:
bd5485dde517d972244713b8eef7d8a6f78ff268